config/config-mattermod.default.json

```diff
@@ -83,6 +83,7 @@
     "SignedCLAURL": "",
     "NeedsToSignCLAMessage": "",
     "BlacklistPaths": [],
```

Comment:
Maybe to make it more clear we could name it something like BlacklistPathsBase?

Reply:
I was asked by @jupenur to keep this PR backwards compatible. I'm 0/5 on this topic.

Reply:
I don't care about the naming in particular, but @agnivade had some concerns regarding deployment.

Reply:
I initially thought that the new key would replace the old one, which would have been a compatibility break. But Elisabeth is suggesting to rename the config param, which is a different thing, but yet another compatiblity break. I am 0/5 too.

Reply:
I think it's worth the naming cleanup here. `BlockListPathsGlobal` and `BlockListPathsPerRepo` is want I went with.